files/zh-cn/web/css/css_transitions/using_css_transitions/index.md

```diff
@@ -5,103 +5,99 @@ slug: Web/CSS/CSS_transitions/Using_CSS_transitions
 
 {{CSSRef}}
 
-**CSS 过渡**提供了一种在更改 CSS 属性时控制动画速度的方法。其可以让属性变化成为一个持续一段时间的过程，而不是立即生效的。比如，将一个元素的颜色从白色改为黑色，通常这个改变是立即生效的，使用 CSS 过渡后该元素的颜色将逐渐从白色变为黑色，按照一定的曲线速率变化。这个过程可以自定义。
+**CSS 过渡**提供了一种在更改 CSS 属性时控制动画速度的方法。其可以让属性变化成为一个持续一段时间的，而不是立即生效的过程。比如，将一个元素的颜色从白色改为黑色，通常这个改变是立即生效的，使用 CSS 过渡后该元素的颜色将按照一定的曲线速率从白色变化为黑色。这个过程可以自定义。
 
 通常将两个状态之间的过渡称为**隐式过渡**，因为开始与结束之间的状态由浏览器决定。
 
-![A CSS transition tells the browsder to draw the intermediate states between the initial and final states, showing the user a smooth transitions.](/files/4529/TransitionsPrinciple.png)
+![CSS 过渡告诉浏览器绘制初始和最终状态之间的中间状态，向用户展示一个平滑的过渡](transitionsprinciple.png)
 
-CSS 过渡可以决定哪些属性发生动画效果 (明确地列出这些属性)，何时开始 (设置 delay），持续多久 (设置 duration) 以及如何动画 (定义*timing function*，比如匀速地或先快后慢)。
+CSS 过渡可以决定哪些属性发生动画效果（通过*明确地列出这些属性*），何时开始（通过设置 _delay_），持续多久（通过设置 _duration_）以及如何动画（通过定义*时间函数*，比如线性或先快后慢）。
```

Comment:
Hi @jasonren0403, could you update the term "timing function", it's renamed to "easing function". See: mdn/content#27315

Reply:
PR raised https://github.com/mdn/content/pull/27811